implementations/elixir/ockam/ockam_hub/test/hub/service/alias_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Test.Hub.Service.AliasTest do
+  use ExUnit.Case
+
+  alias Ockam.Hub.Service.Alias, as: AliasService
+  alias Ockam.Hub.Service.Echo, as: EchoService
+
+  alias Ockam.Router
+
+  test "alias test" do
+    {:ok, _worker, worker_address} =
+      Test.Hub.Service.AliasTestWorker.start_link(address: {192, 168, 2, 1})
```

Comment:
We usually use binaries for local addresses. Although it's possible to use tuples, this test might be confusing.